README.md

```diff
@@ -52,21 +45,5 @@ like to cite it, we would gratefully recommend the following copy:
 > _Delphi Epidata API_.
 > https://github.com/cmu-delphi/delphi-epidata
 
-# Related work
-
- - Cook, Samantha, et al. "Assessing Google flu trends performance in the United
-   States during the 2009 influenza virus A (H1N1) pandemic." PloS one 6.8
-   (2011): e23610.
- - Broniatowski, David A., Michael J. Paul, and Mark Dredze. "National and local
-   influenza surveillance through Twitter: an analysis of the 2012-2013
-   influenza epidemic." (2013): e83672.
- - Dredze, Mark, et al. "HealthTweets. org: A Platform for Public Health
-   Surveillance using Twitter." AAAI Conference on Artificial
-   Intelligence. 2014.
- - Generous, Nicholas, et al. "Global disease monitoring and forecasting with
-   Wikipedia." (2014): e1003892.
- - Hickmann, Kyle S., et al. "Forecasting the 2013–2014 Influenza Season Using
-   Wikipedia." (2015): e1004239.
- - McIver, David J., and John S. Brownstein. "Wikipedia usage estimates
-   prevalence of influenza-like illness in the United States in near real-time."
-   PLoS Comput Biol 10.4 (2014): e1003581.
+See [related work](docs/related_work.md) for a sample of additional work in
```

Comment:
Could consider linking to the COVIDcast API blog post: https://delphi.cmu.edu/blog/2020/10/07/accessing-open-covid-19-data-via-the-covidcast-epidata-api/ as general friendly intro.  Either here or in the COVIDcast API docs?

Reply:
added a link to the blog post higher up in the page, in the COVIDcast section